t/26-controllerrunning.t

```diff
@@ -85,9 +85,17 @@ subtest streaming => sub {
         ok $size > (10 * 1024), 'test file size is greater than 10 * 1024';
         like $controller->tx->res->content->{body_buffer}, qr/data: \["A\\n"\]/, 'body buffer contains "A"';
         Mojo::IOLoop->one_tick;
+
+        $controller->streamtext('test.txt');
+        $t_file->remove();
+        combined_like { Mojo::IOLoop->one_tick }
+        qr/Connection to worker .* will be closed because \/tmp\/.+\/test.txt changed/,
```

Comment:
I encourage everyone to use different regex delimiters to avoid backslash nightmares, but not blocking merge
```suggestion
        qr{Connection to worker .* will be closed because /tmp/.+/test.txt changed},
```